src/controllers/trips.controller.js

```diff
@@ -35,19 +38,26 @@ class Trips {
       });
     });
 
-    const trip = await db.trip.create({
+    const requestId = await createRequest(currentUser.userId, 'single');
+    await db.trip.create({
       leavingFrom,
       goingTo,
       travelDate,
       returnDate,
       reason,
       hotelId,
+      requestId,
       userId: currentUser.userId,
-      status: 'pending',
-      type: 'return'
+      type
+    });
```

Comment:
@b0nbon1 is working on creating a trip service where this database call could be abstracted from the controller.

Reply:
That's great, Am going to work with him so that this will also be part of that service